ubi_manifest/worker/tasks/depsolver/ubi_config.py

```diff
@@ -10,17 +10,17 @@ class UbiConfigLoader:
 
     def __init__(self, url: str) -> None:
         self._url: str = url
-        self._config_map: dict = {}
-        self._all_config: List[ubiconfig.UbiConfig] = None
+        self._config_map: dict[tuple[str, str, str], ubiconfig.UbiConfig] = {}
+        self._all_config: Optional[list[ubiconfig.UbiConfig]] = None
 
     @property
-    def all_config(self) -> List[ubiconfig.UbiConfig]:
+    def all_config(self) -> list[ubiconfig.UbiConfig]:
         if self._all_config is None:
             self._all_config = self._load_all()
 
         return self._all_config
 
-    def _load_all(self) -> List[ubiconfig.UbiConfig]:
+    def _load_all(self) -> ubiconfig.UbiConfig:
```

Comment:
Yes, I got confused, because it actually returns `Any` and the error was `Returning Any from function declared to return "list[Any]"` so I wrongly removed the `list` part. I think it is because the `ubi-config` module itself is not typed.